### DISCUSSION TITLE
Welcome to go-boilerplate Discussions!

### DISCUSSION THREAD

**Discussion by dextercodo** [codoworks/go-boilerplate#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## Contribution
Feel free to start a new discussion, submit a new PR, make a feature request or etc.. If you would like to join the team, reach out to us on Discord. We are always looking for contributors!

## Contacts
- [Join our Discord channel](https://discord.gg/Q27kgPVub7)
- [Dexter Codo](mailto:dexter@dexterexplains.com)

**Comment by newmizanur** [comment]
Hi, Any plan to integrate github.com/sarulabs/di and sqlx?

**Reply by dextercodo** [reply]
I don't see that coming to the roadmap anytime soon.
